react-native/react/native/pinentry-listeners.desktop.js

```diff
@@ -47,6 +47,9 @@ export default {
         response.result(arg)
         console.log('Sent passphrase back')
       }
+      ipc.removeAllListeners('pinentryNeedProps')
```

Comment:
@MarcoPolo Since it's an `ipc.on(..)` handler I can't check the window easily, but I can check that the handler function is the one we created for this request and that should do the same thing.  It's in commit 838f11 now, what do you think?